packages/@react-aria/list/intl/ar-AE.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Intl files from `@react-aria/grid`. They power the code copied from `useHighlightSelectionDescription` which isn't exported from that package. That hook should probably get moved to `react-aria/selection` since it can be used outside of a grid

Reply:
yeah, it just started its life there, we can move it wherever it makes sense